src/globus_cli/commands/gcs/endpoint/role/create.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+import typing as t
+import uuid
+
+import click
+import globus_sdk
+
+from globus_cli.commands.gcs.endpoint.role._common import role_fields
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, endpoint_id_arg
+from globus_cli.termio import TextMode, display
+from globus_cli.utils import resolve_principal_urn
+
+_VALID_ROLES = ["administrator", "activity_manager", "activity_monitor"]
```

Comment:
Just as a cute idea, how do you like this?

```suggestion
_VALID_ROLES_T = t.Literal["administrator", "activity_manager", "activity_monitor"]
_VALID_ROLES: tuple[str, ...] = t.get_args(_VALID_ROLES_T)
```

Reply:
Sure